addon/extend-qunit.js

```diff
@@ -42,3 +45,13 @@ QUnit.assert.notInclude = function(actual, expected, message) {
 
   this.ok(actualString.indexOf(expected) === -1, message);
 };
+
+assert.hasClass = function(actual, expected, message) {
+  if (!message) {
```

Comment:
What do you think about adding the classes to this message? That would make debugging helpful.